WalletWasabi.Gui/Program.cs

```diff
@@ -39,11 +31,23 @@ private static async Task Main(string[] args)
 				AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
 				TaskScheduler.UnobservedTaskException += TaskScheduler_UnobservedTaskException;
 
-				runGui = await CommandInterpreter.ExecuteCommandsAsync(Global, args);
+				runGui = CommandInterpreter.ExecuteCommandsAsync(Global, args).Result;
```

Comment:
In this cases we use:
```suggestion
				runGui = CommandInterpreter.ExecuteCommandsAsync(Global, args).GetAwaiter().GetResult();
```

Our contribution.md document says we should never use that but it is better that `.Result` because in case of exception `.Result` wraps it around an `AggregateException` while `.GetAwaiter().GetResult()` throws the real exception directly.